### PR TITLE
easydict pr

### PR DESCRIPTION
**Pull request by zhaofeng-shu33**

### CODE REVIEW

File: recipes/easydict/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        \ @Chronos-Sk)\n\n1.6 (2015-01-27)\n================\n\n* Allow setting attributes\
+        \ via setting items (thanks phivos)\n\n1.5 (2014-08-07)\n================\n\
+        \n* Allow subclassing using self instead of class name (thanks Steve Engledow)\n\
+        \n1.4 (2011-03-13)\n================\n\n* Access class attributes like instance\
+        \ attributes\n\n1.3 (2012-02-08)\n================\n\n* Better documentation\
+        \ and tests\n\n1.2 (2011-06-08)\n================\n\n* Fix inclusion of README\n\
+        \n1.1 (2012-04-21)\n================\n\n* Switch to distutils2\n\n1.0 (2011-04-18)\n\
+        ================\n\n* Initial working version"
+    home: https://github.com/makinacorpus/easydict
+    license: LPGL
```
Comment (beckermr) [inline recipes/easydict/meta.yaml]: This should be `LGPL`.

File: recipes/easydict/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    license_family: OTHER
```
Comment (beckermr) [inline recipes/easydict/meta.yaml]: Remove this line.

File: recipes/easydict/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+requirements:
+    host:
+        - python
+        - pip
+    run:
+        - python
+test:
+    imports:
+        - easydict
+about:
```
Comment (beckermr) [inline recipes/easydict/meta.yaml]: You need to add a `license_file` entry to the about section with the name of the license file. This is required by the LGPL license.

File: recipes/easydict/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
+{% set version = "1.9" %}
+package:
+    name: easydict
+    version: {{ version }}
+source:
+    sha256: 3f3f0dab07c299f0f4df032db1f388d985bb57fa4c5be30acd25c5f9a516883b
+    url: https://pypi.io/packages/source/e/easydict/easydict-{{ version }}.tar.gz
+build:
```
Comment (beckermr) [inline recipes/easydict/meta.yaml]: This package can probably be made `noarch: python`. See the docs here: https://conda-forge.org/docs/meta.html#building-noarch-packages

File: recipes/easydict/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    host:
+        - python
+        - pip
+    run:
+        - python
+test:
+    imports:
+        - easydict
+about:
+    description: "*EasyDict* allows to access dict values as attributes (works recursively).\
```
Comment (beckermr) [inline recipes/easydict/meta.yaml]: You should be able to use YAML `|`-syntax to clean this up and remove a lot of the escaping, line continuation, and newline characters.

Comment (zhaofeng-shu33) [reply recipes/easydict/meta.yaml]: Thank you very much.

**Pull request merged**